modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -447,7 +447,8 @@ TEST_P(GAPI_Streaming, SmokeTest_VideoConstSource_NoHang)
     auto testc = cv::GComputation(cv::GIn(in, in2), cv::GOut(out))
         .compileStreaming(cv::GMatDesc{CV_8U,3,cv::Size{256,256}},
                           cv::GMatDesc{CV_8U,3,cv::Size{768,576}},
-                          cv::compile_args(cv::gapi::use_only{getKernelPackage()}));
+                          cv::compile_args(cv::gapi::use_only{getKernelPackage()},
+                                           cv::gapi::queue_capacity{1}));
```

Comment:
Is it possible to test this case in other streaming examples? I believe having a queue capacity of 1 may highlight some synchronization/deadlock problems which were masked previously.

Reply:
Agree, but don't know how to re-use it, maybe make sense to add the new one ?

Reply:
Done

Reply:
So why you removed this one?

ADD: Oh it seems that `capacity{1}` was added to a specific test BODY but not a `VALUES` combinator of `gtest`. So the question is - can you extend one or two existing streaming tests with VALUES of different `queue_capacity` (including no capacity specified at all)? You can make it just by introducing a `cv::GCompileArgs` parameter to a test fixture.

Reply:
Done

Reply:
`cv::GCompileArgs` isn't a good way from my point of view, because you need to write most of combinations manually.

Reply:
Done